### PR TITLE
Visitor can view startpage

### PR DESCRIPTION
**Pull request by psnoya**
[VISITOR can view a start page](https://www.pivotaltracker.com/story/show/176626671)
### User Story
```
As a visitor
In order to understand what weCare is
I want to be able to visit a start page
```
## Changes proposed in this pull request:
* Adding Footer component
* Adding MainPageDisplay component
*  Adding MenuHeader component
* Adding form to the page with semantic-up grid
## Packages/Gems added
*semantic UI
*react-router-dom
## Screenshots (Client)
jdfhkjsd

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  <footer>
+<p  
+id='footer'
+> weCare Food-bag Charity
+</p>
+
+  </footer>
+)
+
+export default Footer
```
Comment (tochman) [inline src/components/Footer.jsx]: Please format your code

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+
+const Footer = () => (
+  <footer>
+<p  
+id='footer'
```
Comment (tochman) [inline src/components/Footer.jsx]: Can you explain the usage for this id?

File: src/App.jsx [hunk 3]
```diff
@@ -1,12 +1,24 @@
 import React from 'react'
-import {useSelector} from 'react-redux'
+import { useSelector } from 'react-redux'
+import MenuHeader from './components/MenuHeader'
+import Footer from './components/Footer'
+// import "semantic-ui-css/semantic.min.css";
```
Comment (tochman) [inline src/App.jsx]: please remove comments

File: src/App.jsx [hunk 4]
```diff
@@ -1,12 +1,24 @@
...
 const App = () => {
   const message = useSelector(state=> state.greeting)
   return (
     <>
+      <MenuHeader 
+      />
+      <MainPageDisplay />
       <h1>{message}</h1>
+      <Footer 
+      color='#ebd2be'/>
```
Comment (tochman) [inline src/App.jsx]: I see that the color is taking effect, but why is this property set here?

File: src/components/Footer.jsx (hunk 2 above)
Comment (psnoya) [reply src/components/Footer.jsx]: None. My bad. typed it on automatic. Is now removed

File: src/components/MainPageDisplay.jsx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+          size='small'
+          circular
+          
+          position='centered'
+          color='#e5a4cb'>Register Here</Button>  
+          <Button secondary
+          circular
+          size='small'
+          position='middle'
+          color='#e5a4cb'>Login Here</Button>
```
Comment (tochman) [inline src/components/MainPageDisplay.jsx]: Please format your code

File: src/components/MainPageDisplay.jsx [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+          
+          <Image  src='./img/wecare_logo.png'/>
+        </Grid.Column>
+      </Grid.Row>
+
+    </Grid>
+  )
+}
+
+export default MainPageDisplay
```
Comment (tochman) [inline src/components/MainPageDisplay.jsx]: There's a lot of empty lines in this file and formatting is off. Hard to read.

File: src/index.js [hunk 7]
```diff
@@ -4,12 +4,16 @@ import App from './App'
...
+import { BrowserRouter } from "react-router-dom"
+import 'semantic-ui-css/semantic.min.css'
 
 window.store = store
 
 ReactDOM.render(
   <Provider store={store}>
+    <BrowserRouter>
     <App />
+    </BrowserRouter>
```
Comment (tochman) [inline src/index.js:16]: Please mind the formatting.

File: src/App.jsx (hunk 4 above)
Comment (psnoya) [reply src/App.jsx]: Fixed. Color is now baking taken only from the css file.

**Pull request merged**